### PR TITLE
updated the table in the pdf to improve accessibility

### PR DESCRIPTION
**Pull request by Yasmin-jones**

### CODE REVIEW

File: app/upw/templates/pdf-preview-and-declaration/components/pdf-template/template.njk [hunk 1]
```diff
@@ -752,22 +752,16 @@
...
-                <tr>
-                    <th class="upw-pdf-chart__cell--mid-grey upw-pdf-chart__cell--border-top upw-pdf-chart__cell--border-right" rowspan="2">Number of hours to complete</th>
-                    <th class="upw-pdf-chart__cell--mid-grey upw-pdf-chart__cell--border-top upw-pdf-chart__cell--border-right">1 day per week</th>
-                    <th class="upw-pdf-chart__cell--mid-grey upw-pdf-chart__cell--border-top upw-pdf-chart__cell--border-right">2 days per week</th>
-                    <th class="upw-pdf-chart__cell--mid-grey upw-pdf-chart__cell--border-top upw-pdf-chart__cell--border-right">3 days per week</th>
-                    <th class="upw-pdf-chart__cell--mid-grey upw-pdf-chart__cell--border-top">4 days per week</th>
-                </tr>
-                <tr>
-                    <td class="upw-pdf-chart__cell--light-grey upw-pdf-chart__cell--border-top" colspan="4">…they will complete their hours in this many weeks*</td>
+                    <th class="upw-pdf-chart__cell--mid-grey upw-pdf-chart__cell--border-top upw-pdf-chart__cell--border-right">Number of hours of community payback to complete</th>
```
Comment (LucasCairns) [inline app/upw/templates/pdf-preview-and-declaration/components/pdf-template/template.njk:760]: This table heading doesn't have paragraph tags around it's content, just wanting to check this is intentional?

**Review by LucasCairns — APPROVED** [review]
LGTM 👍

File: app/upw/templates/pdf-preview-and-declaration/components/pdf-template/template.njk (hunk 1 above)
Comment (Yasmin-jones) [reply app/upw/templates/pdf-preview-and-declaration/components/pdf-template/template.njk:760]: Yes it was intentional, following the figma designs where this heading is to stay in bold but the other headings only certain text was to be in bold.

Comment (LucasCairns) [reply app/upw/templates/pdf-preview-and-declaration/components/pdf-template/template.njk:760]: 👍

**Pull request merged**